.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 'iojs-v1'
```

Comment:
Actually, travis just uses [nvm](http://nvm.sh). This is an alias in nvm for any minor and patch versions of iojs 1. It'll grab whatever is the latest.